clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -62,6 +65,46 @@ def __init__(self, cluster: str, pool: Optional[str]) -> None:
         )
         self._nodes_by_ip = {}
 
+    def _cordon_node(self, hostname: str) -> None:
+        self._core_api.patch_node(
+            name=hostname, body={"spec": {"unschedulable": True,},},
+        )
+
+    def _list_all_pods_on_node(self, hostname: str) -> List[KubernetesPod]:
+        return self._core_api.list_pod_for_all_namespaces(field_selector=f"spec.nodeName={hostname}")
+
+    def _evict_tasks_from_node(self, hostname: str) -> None:
+        pods_to_evict = [
+            pod
+            for pod in self._list_all_pods_on_node(hostname)
+            if not self._pod_belongs_to_daemonset(pod) and pod.status and pod.status.phase == "Running"
```

Comment:
This will ignore pods which scheduled right before "cordon", Because they will be "Pending" phase for a while.